doc/00DME0BRD.md

```diff
@@ -0,0 +1,53 @@
+# Business Requirements Document
+
+## Project overview
+
+One of the first elements of a  Modern Operating System (OS) that is perceived by any type of user is the usability of its interaction environment (either command line or graphical user interface based).
+
+Regardless the technologies implied in the general architecture of the OS or the number of applications it can be used for, the we can interact with it is the very first element a user will always perceive.
```

Comment:
```suggestion
Regardless of the technologies implied in the general architecture of the OS or the number of applications it can be used for, how we can interact with it is the very first element a user will always perceive.
```